RELEASE.md

```diff
@@ -27,10 +27,10 @@
 * Removed global `_active_session`, `_activate_session` and `_deactivate_session`. Plugins that need to access objects such as the config loader should now do so through `context` in the new `after_context_created` hook.
 * `config_loader` is available as a public read-only attribute of `KedroContext`.
 * Made `hook_manager` argument optional for `runner.run`.
-
+* `kedro docs` now opens an online version of the Kedro documentation instead of a locally built version.
```

Comment:
Maybe also good to highlight the changes (greatly reduced package size?).

Reply:
Sounds good will add it in.